drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -21,7 +21,41 @@ matterManufacturer:
     productId: 0x5E
     deviceProfileName: plug-binary
 
+#Bridge devices need manufacturer specific fingerprints until
+#bridge support is released to all hubs. This is because of the way generic
+#fingerprints work for bridges joined prior to hubs being able to support them
```

Comment:
Can you add Philips Hue (0x100B/0x0002) and Switchbot (0x1397/0x07e7) as well?

Reply:
Interesting, the Switchbot product has a device type of 0 in the DCL. Ill add it anyway.

Reply:
Yeah, we just know it's a Matter bridge from their public communication